virtool/samples/api.py

```diff
@@ -273,16 +272,14 @@ async def finalize(req):
 
     sample_id = req.match_info["sample_id"]
 
-    await virtool.samples.db.finalize(
-        req.app["db"],
-        req.app["pg"],
+    await get_data_from_req(req).samples.finalize(
         sample_id,
         data["quality"],
         to_thread,
         get_config_from_req(req).data_path,
     )
 
-    return json_response(await virtool.samples.db.get_sample(req.app, sample_id))
+    return json_response(await get_data_from_req(req).samples.get(sample_id))
```

Comment:
This will look like the following after the change to `finalize()`.
```suggestion
    return json_response(sample)
```